src/giftCards/GiftCardsList/GiftCardsListOrderCard/GiftCardsListOrderCard.tsx

```diff
@@ -0,0 +1,39 @@
+import { Link, makeStyles, Typography } from "@material-ui/core";
+import { Alert } from "@saleor/macaw-ui";
+import { orderGiftCardBoughtPath } from "@saleor/orders/urls";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import { giftCardListOrderCardMessages as messages } from "./messages";
+
+const useStyles = makeStyles(
+  theme => ({
+    alert: {
+      marginTop: theme.spacing(2)
+    }
+  }),
+  { name: "GiftCardsListOrderCard" }
+);
+
+const GiftCardsListOrderCard: React.FC = () => {
+  const classes = useStyles();
+
+  return (
+    <Alert variant="info" close={false} className={classes.alert}>
+      <Typography>
+        <FormattedMessage
```

Comment:
These are two separate sentences, one of which is a link. I'd put them here as a FormattedMessage and a Link with FormattedMessage inside of it

Reply:
Actually this should be one message, doesn't matter if it's split in two sentences.
https://formatjs.io/docs/react-intl/components#rich-text-formatting

We can do something like `...message. <link>View Orders with Gift Cards</link>`

Reply:
It was like that initially 😛

Reply:
@dominik-zeglen why

Reply:
Because:
1. We translate messages as a whole, not sentence-by-sentence. This way you retain context of what you are actually translating
2. Current implementation relies on translator putting dot and a space after first sentence. This is unacceptable. On the other hand I don't think it should be joined by `.join(". ")` as I can't guarantee all languages use dot and a space as a sentence separator
3. It's easier and takes care of everything by itself, I can't think of any reason why we shouldn't do it this way

Reply:
Ok, fair enough 👍

Reply:
@dominik-zeglen @bmigirl fixed